### DISCUSSION TITLE
Option to set polling time

### DISCUSSION THREAD

**Discussion by achilleus68** [jobvk/Home-Assistant-Windcentrale#122] [Ideas]
At the moment it seems like the integration polls every minute. I am not sure if that is needed. Polling every e.g. 5 minutes would be sufficient?
Maybe add an option to set the polling time out?

**Comment by jobvk** [comment]
You can change it yourself if you change this this [constant](https://github.com/jobvk/Home-Assistant-Windcentrale/blob/master/custom_components/windcentrale/const.py#L19) to a 5.

The API does indeed update every 5 min. But if you set the interval to 5 min then you may find that exactly just before that 5 min. And then the value may have been different for several minutes.

**Comment by achilleus68** [comment]
Ok, thanks!
